packages/shared/src/lib/featureManagement.ts

```diff
@@ -18,6 +22,10 @@ const feature = {
   lowImps: new Feature('feed_low_imps'),
   bookmarkOnCard: new Feature('bookmark_on_card', false),
   feedLayout: new Feature('feed_layout', FeedLayout.Control),
+  readingStreaks: new Feature(
+    'reading_streaks',
+    ReadingStreaksExperiment.Control,
+  ),
```

Comment:
nice, you knocked out [this ticket](https://dailydotdev.atlassian.net/browse/MI-77) as well with these changes (almost, if you add a hook as well)

Reply:
Well. I will add the hook then 😂